src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -29,25 +29,33 @@ namespace Microsoft.ML.Runtime.Data
     /// <include file='doc.xml' path='doc/members/member[@name="PCA"]/*' />
     public sealed class PcaTransform : OneToOneTransformBase
     {
+        public static class Defaults
```

Comment:
>Defaults [](start = 28, length = 8)

Is there a particular reason to make this public?

Reply:
After moving the file into the same project, I am making it `internal`. I am using it in `PcaEstimator` class for managing defaults.

---
In reply to: [220046677](https://github.com/dotnet/machinelearning/pull/1017#discussion_r220046677) [](ancestors = 220046677)